internal/resources/konnectivity/agent.go

```diff
@@ -102,7 +102,6 @@ func (r *Agent) mutate(ctx context.Context, tenantControlPlane *kamajiv1alpha1.T
 	return func() error {
 		logger := log.FromContext(ctx, "resource", r.GetName())
 
-		address, _, err := tenantControlPlane.AssignedControlPlaneAddress()
```

Comment:
This must be kept, otherwise `args["--proxy-server-host"] = address` would fail due to missing declaration.

Reply:
Sorry... that was a miss in reverting the previous commit.

Reply:
Fixed now.